db/schema/07_orders.sql

```diff
@@ -0,0 +1,12 @@
+DROP TABLE IF EXISTS orders CASCADE;
+
+CREATE TABLE orders (
+
+  id SERIAL PRIMARY KEY NOT NULL,
+  user_id INTEGER REFERENCES users(id) ON DELETE CASCADE,
+  restaurant_id INTEGER REFERENCES restaurants(id) ON DELETE CASCADE,
+  total DECIMAL(7,2) NOT NULL DEFAULT 0,
+  date DATE,
```

Comment:
Should we use `TIMESTAMP` instead `DATE`?

Reply:
Will do! For some reason I thought the timestamp was on the query side.